tests/tpch/dask_queries.py

```diff
@@ -570,19 +571,20 @@ def query_11(dataset_path, fs):
     ).merge(nation, left_on="s_nationkey", right_on="n_nationkey", how="inner")
     joined = joined[joined.n_name == "GERMANY"]
 
-    threshold = ((joined.ps_supplycost * joined.ps_availqty).sum() * 0.0001).compute()
+    threshold = (joined.ps_supplycost * joined.ps_availqty).sum() * 0.0001
 
-    def calc_value(df):
-        return (df.ps_supplycost * df.ps_availqty).sum().round(2)
+    joined["value"] = joined.ps_supplycost * joined.ps_availqty
 
-    return (
-        joined.groupby("ps_partkey")
-        .apply(calc_value, meta=("value", "f8"))
+    res = joined.groupby("ps_partkey").value.sum()
```

Comment:
.value is dangerous, because pandas has attributes that are very very similar which would take precedence, I would prefer square brackets here or a different name